module_1/task_1/index.js

```diff
@@ -0,0 +1,3 @@
+import { ReverseTransform } from "./ReverseTransform";
+
+process.stdin.pipe(new ReverseTransform()).pipe(process.stdout);
```

Comment:
Good solution. JFY you could also use https://nodejs.org/api/readline.html#readline_readline